### DISCUSSION TITLE
How To Play - Still working on headers

### DISCUSSION THREAD

**Discussion by Bruce-Browninggg** [aeolus-1/idlenoughts#10] [Ideas]
<bold> Intro </bold>
Welcome to Idle Noughts, the worlds first Idle TicTacToe game. In Idle Noughts you play TicTacToe to get money called ‘Tics’, use Tics to buy AI, use Ai to play more, play more to get more Tics, use more Tics to get more upgrades. Every time you lose or tie you get a tac, save Tacs for later.

<bold> AI </bold>
In Idle Noughts you can buy AIs to do your work, you lazy fuck. There are different AI’s who act differently. You start with a Jai bot. Each bot has increasing intelligence, except Jai bot who doesn’t have a brain because he is retarted. In the tool bar you can upgrade your bots to play faster and learn faster. Learning faster means they will win more games. If your bot isn’t satisfactory you can recycle it for spare Tics, I recommend recycling Jai bot because it is a needy whore.

<bold> Board </bold>
To progress, buy more boards. There are three types of boards with different advantages. We know you are a selfish bastard so you can buy as many boards as you like. Upgrades are listed on the board and to delete a board use the button in the top right. To beat the game delete the first board.

<bold> Tics, Tacs and Toes </bold>
Every time you lose a game you get a Tac, nobody wants Tacs because it means you suck dick, especially if you are using Jai bots. Luckily for your ungrateful ass you can buy Converter AIs which convert Tacs into something useful such as Tics or Toes.

This game is pretty kinky so you can collect Toes which you can use as a currency in the Toe Shop to unlock bonuses and upgrades. You can also upgrade your Tier to unlock more.

Earn Tics to get a spot on the leader board and battle to be the biggest Nerd, good luck losers.

*DISCLAIMER*
We love Jai <3

**Comment by aeolus-1** [comment]
headers now

**Comment by Bruce-Browninggg** [comment]
This guy deserves a raise
